modules/apps/asset/asset-categories-test/src/testIntegration/java/com/liferay/asset/categories/service/test/AssetVocabularyServiceTest.java

```diff
@@ -549,9 +601,25 @@ protected int searchCount() throws Exception {
 		return results.getLength();
 	}
 
+	private void _testAssetCategoryLongTitlesAreTrimmed(
+		AssetVocabulary assetVocabulary, String title) {
+
+		Map<Locale, String> titleMap = assetVocabulary.getTitleMap();
+
+		for (Map.Entry<Locale, String> entry : titleMap.entrySet()) {
+			Assert.assertEquals(title, entry.getValue());
+		}
+	}
```

Comment:
Can you rename this to `_assertAssetCategory...`? This looks like an assertion method, not a full blown test.

Reply:
done!

Thanks!